zarr/tests/test_storage.py

```diff
@@ -1144,10 +1165,10 @@ def test_chunk_nesting(self):
         # any path where last segment looks like a chunk key gets special handling
         store['0.0'] = b'xxx'
         assert b'xxx' == store['0.0']
-        assert b'xxx' == store['0/0']
+        # assert b'xxx' == store['0/0']
```

Comment:
Why did we loose there? Doesn't the store normalise "." -> "/" anyway?

Reply:
I tried to describe in the commit message on https://github.com/zarr-developers/zarr-python/pull/773/commits/e1835667116b3f86fe002183a6b527565743795c, at least to the best of my understanding.